posix.mak

```diff
@@ -269,12 +269,10 @@ dlangspec.tex : $(DMD) $(DDOC) latex.ddoc dlangspec-consolidated.d
 	$(DMD) -Df$@ $(DDOC) latex.ddoc dlangspec-consolidated.d
 
 # Run twice to fix multipage tables and \ref uses
-dlangspec.dvi : dlangspec.tex
-	latex $^
-	latex $^
-
-$(DOC_OUTPUT_DIR)/dlangspec.pdf : dlangspec.dvi
-	dvipdf $^ $@
+$(DOC_OUTPUT_DIR)/dlangspec.pdf : dlangspec.tex
+	pdflatex -draftmode $^
+	pdflatex $^
+	-mv dlangspec.pdf $(DOC_OUTPUT_DIR)/dlangspec.pdf
```

Comment:
Then let's remove the intermediate junk. Clearly we can't afford to silently ignore a failure. Possibility: generate in `/tmp/` then move.